src/commands/kv/delete_key.rs

```diff
@@ -0,0 +1,26 @@
+use cloudflare::framework::apiclient::ApiClient;
+
+use cloudflare::endpoints::workerskv::delete_key::DeleteKey;
+
+use crate::terminal::message;
+
+pub fn delete_key(id: &str, key: &str) -> Result<(), failure::Error> {
+    let client = super::api_client()?;
+    let account_id = super::account_id()?;
+
+    let msg = format!("Deleting key \"{}\"", key);
+    message::working(&msg);
+
+    let response = client.request(&DeleteKey {
+        account_identifier: &account_id,
+        namespace_identifier: id,
+        key: key,
```

Comment:
if it doesn't happen in https://github.com/cloudflare/cloudflare-rs/pull/29, we need to percent encode here.

Reply:
I think the percent encoding logic should live in wrangler, since we'll be able to have more flexibility over it, plus cloudflare-rs (I think) is supposed to be a thin layer over the API--what do you think?

Reply:
resolution: percent encoding logic now lives in cloudflare-rs :)